### ISSUE TITLE
How to request features?

### ISSUE THREAD

**Issue by cbellosoto** [appstome/timetome-app#20]
Is there a way to request features or provide ideas if you are taking any?

**Comment by appstome** [comment]
Yes, of course. Feel free to create issues or email hi@timeto.me. I don't guarantee this will be implemented, but in any case I will consider or suggest an alternative way.
